trunk/SUAVE/Analyses/Mission/Segments/Climb/Optimized.py

```diff
@@ -64,9 +66,9 @@ def __defaults__(self):
         #   Initialize - before iteration
         # --------------------------------------------------------------
         initialize = self.process.initialize
-        
         initialize.expand_state            = Methods.expand_state
-        initialize.unknowns                = Methods.Climb.Optimized.initialize_unknowns
+        initialize.solved_mission          = Methods.Climb.Optimized.solve_linear_speed_constant_rate
+        #initialize.unknowns                = Methods.Climb.Optimized.initialize_unknowns
```

Comment:
It looks like you deleted this function later, so maybe remove this comment as well.